src/app/components/three-events/three-events.component.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../mixins.scss';
+
+.component-wrapper {
+    @include flex-container(row, flex-start);
+    justify-content: space-evenly;
```

Comment:
the second value for flex-container is justify-content.
use @include flex-container(row, space-evenly);